tests/test_build_latex.py

```diff
@@ -831,7 +831,7 @@ def test_latex_table_tabulars(app, status, warning):
 
     # simple_table
     table = tables['simple table']
-    assert ('\\noindent\\begin{tabulary}{\\linewidth}{|L|L|}' in table)
+    assert ('\\begin{tabulary}{\\linewidth}{|L|L|}' in table)
```

Comment:
I think we should always check `\\begingroup\n\\centering\n\\begin{tabulary}...` here.
It is a part of start tag of both tabular and tabulrary.